CHANGELOG.adoc

```diff
@@ -3,6 +3,8 @@
 
 = UNRELEASED
 
+=== Updated agent-rs to 0.19.0
```

Comment:
Please follow the pattern used by previous versions, where changes like this go in a `Dependencies` section.

Reply:
forgot about adding some info about `ignore` field